python/ray/train/tests/test_huggingface_gpu.py

```diff
@@ -50,13 +54,9 @@ def test_predict_batch(ray_start_4_cpus, caplog, batch_type, device):
     else:
         raise RuntimeError("Invalid batch_type")
 
-    kwargs = {}
-    if device:
-        kwargs["device"] = device
-    with caplog.at_level(logging.WARNING):
-        predictions = predictor.predict(dataset, num_gpus_per_worker=1, **kwargs)
-    assert "enable GPU prediction" not in caplog.text
+    predictions = predictor.predict(dataset, num_gpus_per_worker=1)
 
+    assert "enable GPU prediction" not in caplog.text
```

Comment:
what exactly is this supposed to test...it doesn't look like there's any warning message with this text so this will always be True 😅

Reply:
I'll mock it to ensure the device is set correctly.